designer/settings/base.py

```diff
@@ -108,6 +108,8 @@
 
 TIME_ZONE = 'UTC'
 
+USE_DEPRECATED_PYTZ = True
```

Comment:
Verified

Reply:
Django switched from the `pytz` third-party python package to the Python native package `zoneinfo` in Django 4.0. In release notes, they have mentioned that both are not equivalent so one might need some manual work for this migration. So Django provided this `USE_DEPRECATED_PYTZ` setting, so we can decouple this migration from the Django upgrade.

More info: https://docs.djangoproject.com/en/4.2/releases/4.0/#zoneinfo-default-timezone-implementation